modules/ROOT/pages/salesforce/salesforce-connector-100-studio.adoc

```diff
@@ -36,24 +34,39 @@ Anypoint Studio provides two ways to add the connector to your Studio project: f
 +
-** *On Deleted Object* - Initiate your app when a Salesforce user deletes an object. Possible object types are `Contact`, `Lead`, `Account`, `Contracts`, `Opportunity`, or the name of a custom object.
-** *On Modified Object* - Initiate your app when a Salesforce user modifies an object. Possible object types are `Contact`, `Lead`, `Account`, `Contracts`, `Opportunity`, or the name of a custom object.
-** *On New Object* - Initiate your app when a Salesforce user creates a new object. Possible object types are `Contact`, `Lead`, `Account`, `Contracts`, `Opportunity`, or the name of a custom object.
-** *Replay channel listener* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place in the specified channel trigger the flow.
+* *On Deleted Object*
 +
-NOTE: A channel or topic must be published to Salesforce before a subscription to the channel can be created.
+Initiates your app when a Salesforce user deletes an object. Possible object types are `Contact`, `Lead`, `Account`, `Contracts`, `Opportunity`, or the name of a custom object.
 +
-** *Replay topic listener* - Subscribe to a topic. This trigger provides topic-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place on records in the specified topic trigger the flow.
-** *Subscribe channel listener* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events that take place after you subscribe.
-** *Subscribe topic listener* - Subscribe to a topic. This trigger provides topic-related notification settings for new events that take place after you subscribe.
+* *On Modified Object*
 +
-IMPORTANT: For the *On New Object* and *On Modified Object* operations, specify a UTC value in the *Since* field. If the server is in a different time zone, you must specify a value for the *Since* field; if the *Since* field is left empty, the connector takes the local timestamp. As servers are in datacenters located generally elsewhere from one's location, use of the *Since* field is recommended.
+Initiates your app when a Salesforce user modifies an object. Possible object types are `Contact`, `Lead`, `Account`, `Contracts`, `Opportunity`, or the name of a custom object.
++
+* *On New Object*
+Initiates your app when a Salesforce user creates a new object. Possible object types are `Contact`, `Lead`, `Account`, `Contracts`, `Opportunity`, or the name of a custom object.
+* *Replay channel listener*
++
+Subscribes to a streaming channel. This input source provides channel-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place in the specified channel input source the flow.
++
+* *Replay topic listener*
```

Comment:
I have been assuming but should probably double-check: the UI labels use inconsistent capitalization?